### ISSUE TITLE
Schema file reference did not work with allOf, anyOf, & oneOf

### ISSUE THREAD

**Issue by zaniar** [apioo/fusio#107]
```json
{
  "type": "object",
  "properties": {
    "name": {
      "type": "string"
    },
    "content": {
      "oneOf": [
        { "$ref": "file:///path/to/content_1.json" },
        { "$ref": "file:///path/to/content_2.json" }
      ]
    }
  }
}
```
Will throw the following error when deployed.
```
- [ERROR] schema NameOfTheScema: Unknown protocol scheme file in path-to-my-project/vendor/psx/schema/src/Parser/JsonSchema/RefResolver.php on line 144
```
Since the latest version allowed me to use `schema:///` I tried this:
```json
{
  "type": "object",
  "properties": {
    "name": {
      "type": "string"
    },
    "content": {
      "oneOf": [
        { "$ref": "schema:///Content_1" },
        { "$ref": "schema:///Content_2" }
      ]
    }
  }
}
```
It worked just fine.

**Comment by k42b3** [comment]
Hi, ok I could reproduce the problem, the code which has resolved the `file://` scheme has not traversed through arrays and since the `oneOf` values are inside an array the scheme was not resolved. I have fixed the problem and also improved various other aspects. It is now also possible to access specific elements inside a file i.e. `file:///path/to/content_1.json#/properties/name` to access a specific fragment. If you like you can take a look at the commit here: https://github.com/apioo/fusio-impl/commit/d0c7950e9892fa73f84f0d7d754bf311378bd149

**Comment by k42b3** [comment]
Fixed in the latest v1.0.0-RC6 release
